docs_new/api/processors/structured.md

```diff
@@ -0,0 +1 @@
+::: outlines.processors.structured
```

Comment:
Is there a way to have these generated automatically?

Reply:
Maybe using https://pdoc.dev, which we could also run in CI with each PR when the doc is built.

Reply:
These are generated by mkdocs currently, we shouldn't need any changes.

Reply:
But need manual updates, which is less than ideal for an API documentation.

Reply:
I'm not sure what you mean. mkdocs replaces every single one of these with the docstrings in that module -- it is an automatic API documentation generator.